src/api.js

```diff
@@ -42,3 +42,9 @@ export function getTopics() {
     return response.data.topics;
   });
 }
+
+export function deleteComment(id) {
+  return api.delete(`/comments/${id}`).then((response) => {
```

Comment:
You don't need the .then block, just returning the original promise would be enough, you could do this to obfuscate the response but it isn't really necessary as the delete endpoint has no content anyway

Reply:
Good spot! Thanks :)